README.md

```diff
@@ -201,6 +201,27 @@ This section has been eliminated in the Meteor version, because it does not spec
     };
     ```
 
+  - [3.8](#3.8) <a name='3.8'></a> Use object spreads `...` to copy objects.
```

Comment:
I like it, but emacs js2-mode doesn't know how to parse it, so when you use it a random swath of code afterwards starts getting treated weirdly.  I think a reasonably large number of MDG folks use js2-mode. @dgreensp suggested that it's easy to get PRs in though.  (But I dunno if they would take PRs for an experimental feature?)